src/main/java/de/unistuttgart/iste/gits/flashcard_service/persistence/dao/FlashcardProgressDataLogEntity.java

```diff
@@ -0,0 +1,32 @@
+package de.unistuttgart.iste.gits.flashcard_service.persistence.dao;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Entity(name = "FlashcardProgressDataLog")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class FlashcardProgressDataLogEntity {
```

Comment:
Du hast dich also doch dazu entschlossen, einen Log anzulegen? Aber nur in der Datenbank, sehe ich das richtig?